privacyidea/static/components/token/views/token.details.html

```diff
@@ -142,6 +143,18 @@ <h3 name="Token Details"><translate>Token details for {{ tokenSerial }}
             callback-cancel="get()"
             callback="set(key, value)"
             logged-in-user="loggedInUser"></tr>
+        <tr ng-show="loggedInUser.role==='user'"
```

Comment:
We dublicate the row for the description, one time for the admin adnd the other time for the user.
This is simpler, since there are different conditions when the row and the edit button is visible and which function gets called.